tests/cli/test_utils.py

```diff
@@ -323,3 +325,22 @@ def test_validate_assistant_id_in_config(config_file: Text) -> None:
 
     # reset input files to original state
     rasa.shared.utils.io.write_yaml(copy_config_data, config_file, True)
+
+
+def test_validate_assistant_id_in_config_preserves_comment() -> None:
+    config_file = "data/test_config/config_no_assistant_id_with_comments.yml"
+    copy_config_data = copy.deepcopy(rasa.shared.utils.io.read_yaml_file(config_file))
+
+    rasa.cli.utils.validate_assistant_id_in_config(config_file)
```

Comment:
I'd add a comment that we run this function to write the missing assistant_id to the config file and assert that it's present in the config data read from this file.

Reply:
Ok. I felt that assert wasn't particularly necessary in this case, but I can add it.